demo/index.html

```diff
@@ -0,0 +1,183 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<html>
+<head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+  <title>paper-dropdown-menu</title>
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../../iron-icons/iron-icons.html">
+  <link rel="import" href="../../iron-icon/iron-icon.html">
+  <link rel="import" href="../../iron-image/iron-image.html">
+  <link rel="import" href="../../paper-menu/paper-menu.html">
+  <link rel="import" href="../../paper-item/paper-item.html">
+  <link rel="import" href="../../paper-button/paper-button.html">
+  <link rel="import" href="../../paper-styles/demo-pages.html">
+  <link rel="import" href="../../paper-tabs/paper-tabs.html">
+
+  <link rel="import" href="../paper-dropdown-menu.html">
+
+  <style>
+    paper-menu {
+      display: block;
+    }
+
+    paper-dropdown-menu {
+      text-align: left;
+      margin: auto;
+      width: 180px;
+    }
+
+    paper-dropdown-menu.letters {
+      width: 90px;
+    }
+
+    paper-tabs {
+      width: 400px;
+    }
+
+    iron-image {
+      padding: 1em;
+    }
+
+    .horizontal-section {
+      text-align: center;
+    }
+  </style>
+
+</head>
+<body>
+
+
+  <template id="Demo" is="dom-bind">
+
+    <div class="horizontal-section-container">
+      <div>
+        <h4>Basic Menu</h4>
+        <div class="horizontal-section">
+          <paper-dropdown-menu label="Dinosaurs">
+            <paper-menu class="dropdown-content">
```

Comment:
Although very few static things make sense in this element. @morethanreal 's suggestion for alternate content was color picker, which is a good example.